flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -692,6 +692,8 @@ private void connect(Integer headOfChain, StreamEdge edge) {
 		}
 		// set strategy name so that web interface can show it.
 		jobEdge.setShipStrategyName(partitioner.toString());
+		jobEdge.setDownstreamChannelStateRescaler(partitioner.getUpstreamChannelStateRescaler());
+		jobEdge.setUpstreamChannelStateRescaler(partitioner.getDownstreamChannelStateRescaler());
```

Comment:
OK now I see why the Rescalers used seemed inverted to me :)
So why is it inverted here? (maybe add a comment?)

Reply:
There is no good reason except me screwing up. I did a couple of renamings prior to republishing PR and apparently renamed things asymetrically. I adjusted it (so upstream is now set to upstream).
Sorry for the confusion and good catch.

Reply:
No worries, thanks for updating.